phpunit/tests/term-meta.php

```diff
@@ -1,4 +1,4 @@
-<?php
+<?php // phpcs:ignore WordPress.Files.FileName
```

Comment:
This should be added as a exclude rule to the `phpunit.xml.dist` file.

Reply:
PHPCS does not provide a way to exclude rules for certain files only :(

What to do now?

Reply:
It most definitely does.

```xml
<rule ref="WordPress.Files.FileName">
    <exclude-pattern>*/phpunit/tests/*\.php$</exclude-pattern>
</rule>
```

Note: exclude patterns are treated like PCRE regexes with one caveat, that all `*`s are automatically converted to `.*`.

Reply:
Thank you @jrfnl!

Done.